pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Execute a command on a group of hosts:
+
+`ansible {{group}} -m command -a '{{command}}'`
+
+- Execute a command with administrative priviledges:
+
+`ansible {{group}} -bKm command -a '{{command}}'`
```

Comment:
- **b (become):** Run with [administrative priviledges](http://docs.ansible.com/ansible/become.html#become-privilege-escalation)
- **K (ask become pass):** Ask user to input password for sudo (otherwise Ansible will try to sudo using an empty password)

Excerpt from `ansible -h`:

![screen shot 2016-11-16 at 7 09 19 am](https://cloud.githubusercontent.com/assets/15131271/20329644/1a8ca42c-abcc-11e6-8bce-b1d09ebf879a.png)